lib/eventum/class.mime_helper.php

```diff
@@ -177,13 +177,17 @@ public static function encodeAddress($address)
         if (self::is8bit($address)) {
             // split into name and address section
             preg_match('/(.*)<(.*)>/', $address, $matches);
+
+            $qq = preg_replace_callback(
+                '/([\x80-\xFF]|[\x21-\x2F]|[\xFC]|\[|\])/',
+                function ($m) {
+                    return '=' . strtoupper(dechex(ord(stripslashes($m[1]))));
```

Comment:
also, the said regexp is not multi-byte aware. all it is capable is encode non-printable bytes. or can just say latin1 charset, which is single byte width characters.